doc/developer/tests/test-markdown.md

```diff
@@ -1,25 +1,70 @@
-# Markdown tests
+# Markdown Testing
 
-Paste the following tests into Mattermost to test markdown support.
+### Text Style
+
+_This should render as italics_  
+*This should render as italics*  
+**This should render as bold**  
+***This should render as bold-italics***  
+**_This should render as bold-italics_**  
+~~This should render as strikethrough~~  
+
+### Code Blocks
 
 ```
-# This should render as Heading 1 font size 
-## This should render as Heading 2 font size 
-### This should render as Heading 3 font size 
-#### This should render as Heading 4 font size 
-##### This should render as Heading 5 font size 
-###### This should render as Heading 6 font size 
-~~This should show strikethrough formatting~~
-**This should be bold**
+This text should render in a code block
 ```
 
-Here is how it should render: 
+### In-line Code
+
+The word `monospace` should render as in-line code
+
+### In-line Images
+
+This in-line image should render the mattermost/platform build status:  [![Build Status](https://travis-ci.org/mattermost/platform.svg?branch=master)](https://travis-ci.org/mattermost/platform)  
```

Comment:
Please try variations of in-line images with image sizes and file sizes more likely to cause issues with rendering so we can better test corner cases?